src/OTSubscriber.js

```diff
@@ -156,7 +159,9 @@ OTSubscriber.defaultProps = {
   session: null,
   className: '',
   style: {},
-  properties: {},
+  properties: {
+    insertDefaultUI: true,
```

Comment:
See my comment above, I think we don't need to set insertDefaultUI in true by default.

Reply:
ok, I was following opentok documentation and default value to this property is true in SDK, so I don't think it changes anything. but if you suggest I can make changes.